### ISSUE TITLE
Black screen on iMac 12,1 with macOS 12.0.1 after running the new OCLP 0.3.3

### ISSUE THREAD

**Issue by mactito1** [dortania/OpenCore-Legacy-Patcher#810]
### Machine Model

iMac12,1

### Application Version

Latest Build (nightly)

### Application Variant

TUI (Text User Interface)

### Have you verified whether this issue has been opened before? If no, your issue will be closed

true

### Have you verified whether this issue is covered in our guide? ie. Troubleshooting and Legacy Acceleration pages

true

### Is this issue with a 3rd party application?

false

### What versions of macOS are you seeing the problem on?

macOS 12, Monterey

### Where does this issue happen?

Between booting macOS and Login Screen

### What is the Issue?

I had smoothly upgraded my iMac 12,1 from High Sierra to Big Sur with OCLP 0.3.1 and then via OTA to Monterey 12.0.1. Today I downloaded OCLP 0.3.3 and ran it but when I reboot the computer, after the Apple logo the screen remains black. I don't know what to do to fix it. Please help me.

**Comment by khronokernel** [comment]
Assuming you did your did your due diligence, you should have some native version of macOS to return to.

I believe your issue is due to the new spoofless approach default on majority of systems, so this can be resolved by setting a minimal or higher spoof via `Patcher Settings -> SMBIOS Settings -> Set SMBIOS Spoof Level`

However it would be ideal to resolve this for spoofless as well. Could you upload the build folder as well as the build process from the TUI?

Additionally could you elaborate on the hardware of this iMac? Namely the GPU present

**Comment by mactito1** [comment]
Thanks for answering, in relation to what is indicated in your answer I have the USB memory with which I installed Big Sur in my iMac 12,1.
Searching the internet I saw that I could run a recovery mode combining CMD + R to reinstall macOS from the Apple server. But in both cases I am concerned that these actions involve formatting the computer and losing all the information I have on it.

I understand that to change the SMBIOS options in OCLP I must have managed to get macOS to work but as I said after the Apple logo the screen remains black so I don't have how to make that change of options.

My iMac 12,1 has a ATI 6750 512mb GPU and a Intel i5 2400S 2.5Ghz CPU.

Thanks again for your cooperation and I remain attentive to your instructions.

**Comment by mactito1** [comment]
Yesterday I tried to use the macOS recovery method to reinstall Monterey 12.0.1 but when I give it CMD + R the tool only connects to the internet to install Lion (the version of macOS that came from the factory) and I don't get the option to reinstall Monterey.
I don't seem to have a choice if I want to get the computer back up and running.
I'll be commenting on how I'm doing.

**Comment by splaquet** [comment]
I actually ran into something like this today. I had run a pre-release version of 3.2 and was back up and running, but 0.3.3 gave me a black screen.  I wasn't sitting at my computer when it rebooted, so I restarted again...

that's when I realized that it was defaulting to the wrong boot loader in the boot select screen.  

I swapped it over after another reset and loaded up right away.

Probably not, but possibly your case as well.

**Comment by mactito1** [comment]
Hello, finally after many unsuccessful attempts to recover my macOS 12.0.1 installation which had been left on a black screen after executing OCLP 0.3.3 I had to use the pendrive where I have Big Sur and OCLP 0.3.1.
I did the installation without problems and once it was completed the option to upgrade to Monterey 12.1 appeared which I proceeded to do.
When I did it the previous time and it was updated to macOS 12.0.1 the computer was running smoothly but now with macOS 12.1 something is wrong. I clarify that the mouse speed is fine and the computer "does not seem" to be slow but when I open an app it freezes and I must minimize it and maximize it again to "advance".
If it's a macOS Monterey 12.1 theme, I'd better go back to Big Sur.
I await some guidance. Thank you.
